doc/Data Plane Code Generation/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -1,201 +1,168 @@
-- Rename the `Azure.Template.Generated` solution to `Azure.<group>.<service>`.
-- Remove the `Azure.Template.Generated.Tests` project from the solution.
-
-**Note**: `Azure.<group>.<service>` is the Azure SDK package name. The package name/namespace will be something the architects will help standardize according to the guidelines.
-
-**Learn more:** see the [.NET Guidelines on Namespace Naming](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-namespace-naming) for more information on naming Azure SDK packages. This name diverges from approved Azure SDK namespaces, since it is intended to be used just for this tutorial.
-
-#### 4. Update the Assembly Info
+e.g.
+```
+pwsh /home/azure-sdk-for-net/eng/scripts/automation/Invoke-DataPlaneGenerateSDKPackage.ps1 -service sample -namespace Azure.Template.Sample -sdkPath /home/azure-sdk-for-net -inputfiles https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json -securityScope https://sample/.default
+```
 
-You'll notice that the `Azure.<group>.<service>` project has `properties/AssemblyInfo.cs` file in it. Replace `Microsoft.Test` namespace with the correct resource provider namespace for your service.
+**Note**: 
+- Use one of the following pre-approved namespace groups (https://azure.github.io/azure-sdk/registered_namespaces.html): Azure.AI, Azure.Analytics, Azure.Communication, Azure.Data, Azure.DigitalTwins, Azure.IoT, Azure.Learn, Azure.Media, Azure.Management, Azure.Messaging, Azure.ResourceManager, Azure.Search, Azure.Security, Azure.Storage, Azure.Template, Azure.Identity, Microsoft.Extensions.Azure
+- namespace is the shipped package name, it should be Azure.<group>.<service>
+- inputfiles is the api definition files, separated by semicolon if more than one. The api definition file can be local file e.g. ./swagger/compute.json or premlink, e.g. https://github.com/Azure/azure-rest-api-specs/blob/73a0fa453a93bdbe8885f87b9e4e9fef4f0452d0/specification/webpubsub/data-plane/WebPubSub/stable/2021-10-01/webpubsub.json.
 
-**Learn more:** see the [Azure Services Resource Providers](https://docs.microsoft.com/azure/azure-resource-manager/management/azure-services-resource-providers) for the list of possible namespaces.
+- Both AADToken and AzureKey authentication are supported. If your service support AADToken, just set the parameter **securityScope**(the security scope), and if your service support AzureKey authentication, set parameter **securityHeaderName**( the security header name). You also can provide both if your service support two authentications.
 
-#### 5. Save and commit your changes to your branch
+The script `eng\scripts\automation\Invoke-DataPlaneGenerateSDKPackage.ps1` will do **step-by-step** as following.
 
-This will make it easier to see the changes you make when you generate code in the clean solution.
+### Create the project folder
```

Comment:
The way these headers end up in the table of contents, it seems like we're asking the reader to do this, rather than telling them the script will do this.  Would it be possible to adjust the formatting, so we know we're in the context of a script and that these are not actions the reader will take?

Reply:
Curious: would it make sense to link to the script, and then summarize the steps in words, rather than showing the commands that are in the script?

Reply:
update to use list for steps of the Invoke-DataPlaneGenerateSDKPackage.ps1 to avoid confusing